app/server/endpoint/src/main/java/io/syndesis/server/endpoint/v1/handler/integration/support/IntegrationSupportHandler.java

```diff
@@ -277,6 +300,19 @@ private void addResourcesToExport(final JsonDB export, final Integration integra
                     }
                 }
 
+                // import custom icons
+                if( entry.getName().startsWith("icons/") ) {
```

Comment:
nit: funny formatting

Reply:
Agreed but I didn't want to reformat the entire file.